src/test/java/study/Calculator.java

```diff
@@ -0,0 +1,33 @@
+package study;
+
+public class Calculator {
+
+    public static final String DELIMITER = " ";
+
+    public static int calculate(String input) {
+        validate(input);
+
+        String[] values = input.split(DELIMITER);
+
+        InputNumber first = new InputNumber(values[0]);
+
+        for (int i = 1; i < values.length; i+=2) {
+            String operator = values[i];
+            InputNumber second = new InputNumber(values[i+1]);
+
+            first = calculate(first, second, operator);
+        }
+
+        return first.getInputNumber();
+    }
+
+    private static void validate(String input) {
+        if (input == null || input.isEmpty()) {
+            throw new IllegalArgumentException("입력 값이 null이나 빈 공백 문자입니다.");
+        }
+    }
+
+    private static InputNumber calculate(InputNumber first, InputNumber second, String operator) {
+        return Operator.calculate(first, second, operator);
```

Comment:
```suggestion
        return operator.calculate(first, second);
```
꼭 정적메서드를 사용해야하는 것이 아니라면 위와 같은 구조는 어떨까요~?